tools/devtool

```diff
@@ -1507,10 +1518,74 @@ cmd_release() {
             post_release "$version" "$owner" "$repo" "$token" "$prerelease" "$release_text" ||
             delete_remote_tag "$version" "$remote" "Could not post release to GitHub."
             say "Draft release created successful. Go to https://github.com/$owner/$repo/releases to check it out!"
+
+            say "Updating tag for the release with the greatest version..."
+            update_latest_tag "$version" "$remote" "$owner" "$repo" "$token" ||
+            die "Failed to update tag for the release with the greatest version."
     fi
 fi
 }
 
+# GitHub tags releases based on the creation date of their associated tag.
+# When we publish a patch release of an older release, GitHub marks that
+# release with the `latest` label. GitHub's definition of `latest` is
+# `most recent`. However, we want to have `latest` tag on the greatest
+# released Firecracker version, for better visibility for our customers.
+# We need to recreate the tag of the greatest release version, after
+# creating the tag associated with the patch release we want to publish.
+# This way, the tag for the release with the greatest version is also the
+# most recent tag created and GitHub will assign it the `latest` label.
+update_latest_tag() {
+    version="$1"
+    remote="$2"
+    owner="$3"
+    repo="$4"
+    token="$5"
+
+    # List all tags, and fetch the one with the greatest version.
+    greatest_release=$(git tag | grep -E '^v[[:digit:]]+\.[[:digit:]]+\.[[:digit:]]+' | sort -V | tail -1)
+
+    # If this release is the one with the greatest version, GitHub will
+    # automatically assign it the `latest` label. If this is a patch release
+    # of an older version, then we need to recreate the tag of the greatest
+    # version.
+    if [[ ! "$greatest_release" == "v$version" ]]; then
+        say "Recreating tag for the release with the greatest version, so that GitHub labels it as \`latest\`..."
```

Comment:
I am ok with this approach of recreating the tag that points to latest release.

However, since this is quite a sensitive operation, can you add a `get_user_confirmation` before proceeding with the date update?